ydb/core/kqp/opt/kqp_query_plan.cpp

```diff
@@ -2144,6 +2144,47 @@ double ComputeCpuTimes(NJson::TJsonValue& plan) {
     return currCpuTime;
 }
 
+void ComputeTotalRows(NJson::TJsonValue& plan) {
+    
+    if (plan.GetMapSafe().contains("Plans")) {
+        for (auto& p : plan.GetMapSafe().at("Plans").GetArraySafe()) {
+            ComputeTotalRows(p);
+        }
+    }
+
+    if (plan.GetMapSafe().contains("Stats") && plan.GetMapSafe().contains("Operators")) {
+        YQL_CLOG(TRACE, CoreDq) << "Found Operators";
```

Comment:
Let's write more understandable log message or delete it

Reply:
deleted